public/static/locales/de/profile.json

```diff
@@ -33,6 +38,8 @@
       "editTarget": "Ziel bearbeiten"
     },
     "myContributions": {
+      "titlePrivate": "Meine Beiträge",
+      "titlePublic": "{name}'s Beiträge",
```

Comment:
Some people in Germany do not like using `'s` on names as in English as correctly mostly you just append an `s` to the name if possible (unfortunately it's more complicated for different kind of names, so it cannot simply be solved here except using some different wording as `Beiträge von {name}`), compare https://de.pons.com/p/wissensecke/grammatik-to-go/deppenapostroph.